### DISCUSSION TITLE
Add more Adobe Apps.

### DISCUSSION THREAD

**Discussion by dajix350** [YoungFellow-le/photoshop-22-linux#4] [Ideas]
Hi!
First of all, tysm for making this. Until now I have had to dual boot Windows and its a pain in the ass. This makes my life so much easier!

Can you please add more Adobe apps to this installer? Eg Premiere Pro and After Effects? I use them all the time as well and I would assume they would have similar dependencies.

**Comment by albakhtari** [comment]
Hello,
Premiere pro and After Effects have been very problematic for testers, and none have managed to get it to work.
As for the dependencies, then PR and AE would have many more I think.
I used to use PR and AE on a VM (it was a side hobby), but then I swithched to [DaVinci Resolve](https://www.blackmagicdesign.com/products/davinciresolve), it's basically PR+AE+AU+ME, all in one application. And they have a Linux version too.
I would recommend switching over to Resolve.
I have also found that it was much more stable than PR and AE, and it crashed a lot less.
